azure-pipelines/main-pipeline.yml

```diff
@@ -0,0 +1,247 @@
+# The idea is to be inclusive of all source directories. (I.E. Commerce, Accounts, Eureka, Gateway, Inventory, Config, and Base(?))
+# And using the variables to set a condition that will set pipeline variables that, in turn, will control the flow of the pipeline and what builds.
+
+trigger:
+  branches:
+    include: 
+    - main
+    - staging
+    - development
+
+pr:
+  branches:
+    include:
+    - main
+    - staging
+    - development
+    - feature_* 
```

Comment:
inconsistency in yaml indentation here and other places in this file (not blocking)

Reply:
Last push should've solved this.